.github/workflows/release.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
             target: x86_64-pc-windows-msvc
           - os: ubuntu-2004-8-cores
             target: x86_64-unknown-linux-gnu
-          - os: ubuntu-2004-8-cores
+          - os: ubuntu-2204-8-cores
```

Comment:
We might need to create a machine type for this. The existing 8 core ones are custom large runners.

Reply:
how do we do that?

Reply:
<img width="571" alt="Screenshot 2024-01-05 at 1 03 19 PM" src="https://github.com/GlareDB/glaredb/assets/4040560/4b7ba200-286b-434a-b349-5be9a55c37f3">

runner added